java/code/src/com/redhat/rhn/frontend/xmlrpc/chain/ActionChainHandler.java

```diff
@@ -293,6 +293,53 @@ public Integer addErrataUpdate(User loggedInUser,
         return actionIds.get(0).intValue();
     }
 
+    /**
+     * Schedule Errata update.
+     *
+     * @param loggedInUser The current user
+     * @param sids a list of Server IDs
+     * @param errataIds a list of erratas IDs
+     * @param chainLabel Label of the action chain
+     * @return action id if successful, exception otherwise
+     *
+     * @apidoc.doc Adds Errata update to an Action Chain.
+     * @apidoc.param #session_key()
+     * @apidoc.param #array_single_desc("int", "sids", "System IDs")
+     * @apidoc.param #array_single_desc("int", "errataIds", "Errata ID")
+     * @apidoc.param #param_desc("string", "chainLabel", "Label of the chain")
+     * @apidoc.returntype #param_desc("int", "actionId", "The action id of the scheduled action")
+     */
+    public Integer addErrataUpdate(User loggedInUser,
+                                    List<Integer> sids,
+                                    List<Integer> errataIds,
+                                    String chainLabel) {
```

Comment:
Can we change the "single" version from line 269 to call this one ?
We have a lot of duplicate code.